kaldb/src/main/java/com/slack/kaldb/chunkManager/CachingChunkManager.java

```diff
@@ -95,15 +94,17 @@ protected void shutDown() throws Exception {
   public static CachingChunkManager<LogMessage> fromConfig(
       MeterRegistry meterRegistry,
       MetadataStore metadataStore,
-      KaldbConfigs.ServerConfig serverConfig)
+      KaldbConfigs.ServerConfig serverConfig,
+      KaldbConfigs.S3Config s3Config,
+      KaldbConfigs.CacheConfig cacheConfig)
```

Comment:
Why are we passing both `serverConfig` and `cacheConfig`, as `serverConfig` is a property of `cacheConfig`?

Reply:
Good point. Removed serverConfig param.